### ISSUE TITLE
I think its finally dead.

### ISSUE THREAD

**Issue by lecrackzor** [TobiasPankner/Gleam-giveaway-bot#24]
Every entry attempted now gives this error:

Timeout waiting for element. (div[ng-controller='EnterController'])
Timeout waiting for element. (div[ng-controller='EnterController']>div[ng-init^='initCampaign'])

It was fun while it lasted. Thanks Tobias.

**Comment by TobiasPankner** [comment]
Thanks for letting me know.  
I will probably not fix this as I do not support this project anymore.  
If someone wants to create a fork and fix this they are welcome to.

**Comment by Oukeef** [comment]
Oh no :( this makes me very sad. 
If you do decide to update/fix this issue please let me know
Or if anyone decides to work on this and fixes the issue, or find a working one please let me know. Thanks

**Comment by ras4evermore** [comment]
Yes finally Dead :( Need a fix for the recently integrated selenium detection, as well as fix for the added drop down required to see the entries with playrgg
